dtfs-central-api/src/services/repositories/utilisation-report-status-repo.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Collection,
+  DeleteResult,
+  ObjectId,
+  UpdateResult,
+} from 'mongodb';
+import {
+  ReportDetails,
+  ReportFilter,
+  ReportStatus,
+  UtilisationReport,
+} from '../../types/utilisation-report-status';
+import { TFMUser } from '../../types/users';
+
+const REPORT_STATUS: Record<ReportStatus, ReportStatus> = {
+  RECONCILIATION_COMPLETED: 'RECONCILIATION_COMPLETED',
+  REPORT_NOT_RECEIVED: 'REPORT_NOT_RECEIVED',
+};
+
+const logWarningMessage = (status: ReportStatus) => console.error(`The status '${status}' is not supported by '/v1/utilisation-reports/set-status'`);
+
+const setReportStatusByReportId = (id: string, status: ReportStatus, collection: Collection): Promise<UpdateResult | void> => {
```

Comment:
My reason for doing it this way was that I didn't want to make the `db.getCollection` call each time I processed an element of `reportsWithStatus`. I suppose I could instead create an array of sanitised update requests and then provide that array to this file so that the collection logic is the only thing handled here